src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -624,10 +624,20 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             break;
         }
 
-        case NI_Vector64_ConvertToInt32:
         case NI_Vector64_ConvertToInt32Native:
-        case NI_Vector128_ConvertToInt32:
         case NI_Vector128_ConvertToInt32Native:
+        {
+            if (opts.IsReadyToRun())
```

Comment:
I considered making this a flag in the hwintrinsic list table, to allow this to be more easily centralized for any future APIs that fit in this bucket, but opted not to since we don't have many spare bits for the that flags enum right now and this helps keep the cost lower for the main intrinsic path.